app/pages/task-list/taskListManifest.json

```diff
@@ -114,6 +114,34 @@
           ]
         }
       ]
+    },
+    {
+      "name": "Indicate funding source",
+      "sections": [
+        {
+          "id": "funding-sources",
+          "title": "Explain how you're paying for this order",
+          "dependencies": [
+            {
+              "statusDependencies": ["description", "ordering-party", "supplier", "commencement-date", "service-recipients", "catalogue-solutions", "additional-services", "associated-services"]
+            },
+            {
+              "statusDependencies": ["description", "ordering-party", "supplier", "commencement-date", "service-recipients", "associated-services"],
+              "countDependencies": ["associated-services"],
+              "zeroCountDependencies": ["service-recipients"]
+            },
+            {
+              "statusDependencies": ["description", "ordering-party", "supplier", "commencement-date", "service-recipients", "catalogue-solutions", "associated-services"],
+              "countDependencies": ["service-recipients", "associated-services"],
+              "zeroCountDependencies": ["catalogue-solutions"]
+            },
+            {
+              "statusDependencies": ["description", "ordering-party", "supplier", "commencement-date", "service-recipients", "catalogue-solutions", "associated-services"],
+              "zeroCountDependencies": ["catalogue-solutions", "associated-services"]
+            }
```

Comment:
So glad we engineered this in. Imagine code if we did it with just if,elses in the code base

Reply:
yeah defintiely was an easy addition too!